src/Microsoft.ML.TensorFlow/TensorFlowModel.cs

```diff
@@ -27,10 +28,12 @@ public sealed class TensorFlowModel : IDisposable
         /// <param name="env">An <see cref="IHostEnvironment"/> object.</param>
         /// <param name="session">TensorFlow session object.</param>
         /// <param name="modelLocation">Location of the model from where <paramref name="session"/> was loaded.</param>
-        internal TensorFlowModel(IHostEnvironment env, Session session, string modelLocation)
+        /// <param name="treatOutputAsBatched">If the first dimension of the output is unknown, should it be treated as batched or not.</param>
+        internal TensorFlowModel(IHostEnvironment env, Session session, string modelLocation, bool treatOutputAsBatched = true)
```

Comment:
I'm wondering, what is you logic behind making `treatOutputAsBatched`'s default value to be `true`?

Reply:
Most of the time we wanted the output to be batched and that is the current behavior that ML.NET has. So defaulting to true keeps the existing behavior but opens up the choice for the user to change it.